### PR TITLE
x86: remove unsupported cpu models

### PR DESCRIPTION
**Pull request by nanliu-r**
ID: 3037

Remove the deprecated cpu models on different releases since they are not supported any more.

### CODE REVIEW

**Comment by nanliu-r** [comment]
@PaulYuuu Hi, please help review this code, thanks.

**Comment by nanliu-r** [comment]
Test Pass with two latest rhel vm and win2022/win2025 on Intel host and AMD host.

File: qemu/tests/cfg/x86_cpu_flags.cfg [hunk 1]
```diff
@@ -110,7 +110,7 @@
         - others:
             variants:
                 - test_default_model:
-                    no RHEL.9
+                    only RHEL.6, RHEL.7, RHEL.8, Win10, Win2012, Win2016, Win2019
```
Comment (PaulYuuu) [inline qemu/tests/cfg/x86_cpu_flags.cfg:113]: Not need to consider very old EOL guests.

Comment (nanliu-r) [reply qemu/tests/cfg/x86_cpu_flags.cfg]: > Not need to consider very old EOL guests.

@PaulYuuu  Updated, thanks.

**Review by PaulYuuu — APPROVED** [review]

**Pull request merged**